File: kalkulator.py
#membuat fungsi untuk masing-masing operator

#membuat fungsi tambah
def tambah(a,b):
    return a + b

#membuat fungsi kurang
def kurang(a,b):
    return a - b

#membuat fungsi perkalian
def kali(a,b):
    return a * b

#membuat fungsi pembagian
def bagi(a,b):
    return a / b 


#membuat menu operasi
print("pilih operasi")
print("1. penjumlahan")
print("2. pengurangan")
print("3. perkalian")
print("4. pembagian")

#meminta user menginput pilihan
choice = input("masukan pilihan (1): ")
choice = input("masukan pilihan (2): ")
choice = input("masukan pilihan (3): ")
choice = input("masukan pilihan (4): ")

bil1 = int(input("masukan bilangan pertama: "))
bil2 = int(input("masukan bilangan kedua: "))

if choice == "1":
    print(bil1, "+", bil2, "=", tambah(bil1, bil2))

if choice == "2":
    print(bil1, "-", bil2, "=", kurang(bil1, bil2))

if choice == "3":
    print(bil1, "*", bil2, "=", kali(bil1, bil2))

if choice == "4":
    print(bil1, "/", bil2, "=", bagi(bil1, bil2))

File: kalkulator.php
<?php
$a       = $_GET['a'];
$b       = $_GET['b'];
$operasi = $_GET['operasi'];

echo "Nilai y : $a<br>";
echo "Nilai x : $b<br><br>";
 
if ($operasi=="jumlah"){
  $c = $a + $b; // rumus penjumlahan
  echo "Penjumlahan $a + $b = $c";
}
elseif ($operasi=="kurang"){
  $c = $a - $b; // rumus pengurangan
  echo "Pengurangan $a - $b = $c";
}
elseif ($operasi=="kali"){
  $c = $a * $b; // rumus perkalian
  echo "Perkalian $a x $b = $c";
}
elseif ($operasi=="bagi"){
  $c = $a / $b; // rumus pembagian
  echo "Pembagian $a / $b = $c";
}
// apabila operasi perhitungan belum dipilih
else{
  echo "Anda belum memilih operasi perhitungan";
}
?>